theseus_gui/src/components/ui/ExportModal.vue

```diff
@@ -70,6 +72,31 @@ const initFiles = async () => {
     },
     value,
   ])
+
+  if (props.instance.export_cache) {
+    const export_cache = toRef(props.instance.export_cache).value
+    if (export_cache.name) {
+      nameInput.value = export_cache.name
+    }
+    if (export_cache.version) {
+      versionInput.value = export_cache.version
+    }
+    if (export_cache.description) {
+      exportDescription.value = export_cache.description
+    }
+    for (const file of export_cache.overrides) {
+      files.value.forEach((f) => {
+        if (f.path === file) f.selected = true
+      })
+      folders.value.forEach((args) => {
+        args[1].forEach((child) => {
+          if (child.path === file) {
+            child.selected = true
+          }
+        })
+      })
+    }
+  }
```

Comment:
what is the export cache used for? is it really necessary? IMO we should just have better defaults on export.